internal/server/analytics/clickhouse/mutation.go

```diff
@@ -0,0 +1,39 @@
+package clickhouse
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"go.flipt.io/flipt/internal/server/analytics"
+)
+
+// IncrementFlagEvaluationCounts does a batch insert of flag values from evaluations.
+func (c *Client) IncrementFlagEvaluationCounts(ctx context.Context, responses []*analytics.EvaluationResponse) error {
+	if len(responses) < 1 {
+		return nil
+	}
+
+	valuePlaceHolders := make([]string, 0, len(responses))
+	valueArgs := make([]interface{}, 0, len(responses)*9)
```

Comment:
```suggestion
	valueArgs := make([]interface{}, 0, len(responses)*COLUMNS)
```

Could we make `9` a constant so its not a magic number